File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        int answer = 0;
        if (100000 > price && price >= 0){
            answer = price ;
        }else if(300000 > price && price >= 100000){
            answer = (int)((double)price * (1-0.05));
        }else if(500000 > price && price >= 300000){
            answer = (int)((double)price * (1-0.1));
        }else if(price >= 500000){
            answer = (int)((double)price * (1-0.2));
        }
        return answer;
    }
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
def solution(price):
    if 100000 > price >= 0 :
        answer = price 
    elif 300000 > price >= 100000 :
        answer = price * (1-0.05)
    elif 500000 > price >= 300000 :
        answer = price * (1-0.1)
    elif price >= 500000 :
        answer = price * (1-0.2)
        
    return int(answer)

#flout (실수형)